apps/ledger-live-desktop/src/main/index.js

```diff
@@ -111,10 +112,26 @@ app.on("ready", async () => {
     );
   });
 
+  ipcMain.handle("getStoreValue", (event, { key, storeId }) => {
+    const storageKey = storeId ? `${storeId}-${key}` : key;
+    return store.get(storageKey);
+  });
+
+  ipcMain.handle("setStoreValue", (event, { key, value, storeId }) => {
+    const storageKey = storeId ? `${storeId}-${key}` : key;
+    return store.set(storageKey, value);
+  });
+
+  ipcMain.handle("resetStore", event => {
+    return store.clear();
+  });
+
+  /** @deprecated (Use "getStoreValue" instead) */
   ipcMain.handle("getKey", (event, { ns, keyPath, defaultValue }) => {
     return db.getKey(ns, keyPath, defaultValue);
   });
 
+  /** @deprecated (Use "setStoreValue" instead) */
   ipcMain.handle("setKey", (event, { ns, keyPath, value }) => {
     return db.setKey(ns, keyPath, value);
   });
```

Comment:
Can we really consider this deprecated ? All user data are currently stored by the current lib. Moving away from it will require a migration process.

Reply:
i miss a lot of context on what we wanted to do here. do we have documentation on the part that you "noticed buggy" ?

On our side we know we want to consider better solution than app.json with tons of data in it but we would need to brainstorm a bit all solutions & what each would mean.

side note: electron-store used to be a library we were using in the past, and i'm not sure why we moved away from it.